app/jobs/regular/problem_check.rb

```diff
@@ -1,15 +1,29 @@
 # frozen_string_literal: true
 
 module Jobs
+  class RetrySignal < Exception
```

Comment:
This has to inherit `Exception` or it will be gobbled up by the catch-all `rescue StandardError` in `.execute_scheduled_check`.

Reply:
Love comments like this in PRs, because my immediate thought would have been WHY EXCEPTION!??! No panik, only kalm 😌